client/flutter/lib/home_page.dart

```diff
@@ -3,12 +3,15 @@ import 'package:WHOFlutter/generated/l10n.dart';
 import 'package:WHOFlutter/pages/protect_yourself.dart';
 import 'package:WHOFlutter/pages/travel_advice.dart';
 import 'package:WHOFlutter/pages/who_myth_busters.dart';
+import 'package:firebase_analytics/firebase_analytics.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_staggered_grid_view/flutter_staggered_grid_view.dart';
 import 'package:share/share.dart';
 
 class HomePage extends StatelessWidget {
+  final FirebaseAnalytics analytics;
```

Comment:
Can we add a `analytics.logAppOpen()` when the app opens?

Reply:
Firebase automatically logs that, not required.